docs/server/source/cloud-deployment-templates/ca-installation.rst

```diff
@@ -2,81 +2,30 @@ Set Up a Self-Signed Certificate Authority
 ==========================================
 
 This page enumerates the steps *we* use to set up a self-signed certificate authority (CA).
-Your organization might do it differently.
+This is something that only needs to be done once per cluster,
+by the organization managing the cluster, i.e. the CA is for the whole cluster.
+We use Easy-RSA.
```

Comment:
The instructions on the `ca-installation.rst` page, to set up a self-signed CA, only have to be done once per cluster, right? There's only one CA per cluster. I don't understand why you wrote, "and everytime a new `bigchaindb node` is set up." (above)

I understand why there needs to be a different Easy-RSA directory for each of the three possible use cases.